docs/advanced-topics/stack-monitoring.asciidoc

```diff
@@ -70,6 +70,58 @@ You can send metrics and logs to two different Elasticsearch monitoring clusters
 
 You can also enable Stack Monitoring on Elasticsearch only or on Kibana only. In the latter case, Kibana will not be available on the Stack Monitoring Kibana page (check link:https://www.elastic.co/guide/en/kibana/current/monitoring-data.html#monitoring-data[View monitoring data in Kibana]).
 
+== Connect to an external monitoring Elasticsearch cluster
+
+If you want to connect to a monitoring Elasticsearch cluster not managed by ECK, you can reference a Secret instead of an Elastisearch cluster in the `monitoring` section through the `secretName` attribute:
+
+[source,yaml,subs="attributes,callouts"]
+----
+apiVersion: elasticsearch.k8s.elastic.co/{eck_crd_version}
+kind: Elasticsearch
+metadata:
+  name: monitored-sample
+  namespace: production
+spec:
+  version: {version}
+  monitoring:
+    metrics:
+      elasticsearchRefs:
+      - secretName: monitoring-metrics-es-ref <1>
+    logs:
+      elasticsearchRefs:
+      - name: monitoring-logs
+        namespace: observability <2>
+        serviceName: monitoring-logs-es-coordinating-nodes <2>
+  nodeSets:
+  - name: default
+    count: 1
+    config:
+      node.store.allow_mmap: false
+---
+
+<1> The `secretName` and `name` attributes are mutually exclusive, you have to choose one or the other.
+<2> The `namespace` and `serviceName` attributes can only be used in conjunction with `name`, not with `secretName`.
+
+The referenced Secret must contain the following connection information:
+- `url`: the URL to reach the Elasticsearch cluster
+- `username`: the username of the user to be authenticated to the Elasticsearch cluster
+- `password`: the password of the user to be authenticated to the Elasticsearch cluster
+- `ca.crt`: the CA certificate in PEM format to secure communication to the Elasticsearch cluster (optional)
+
+[source,yaml,subs="attributes,callouts"]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: monitoring-metrics-es-ref
+stringData:
+  url: https://mon1.es.abcd-42.xyz.elastic-cloud.com:9243
+  username: monitoring-user
+  password: @j5SiN;<7yLJQz4<7:g:ap7hBb:2slbtfZ}?[PyO
```

Comment:
nit: Even if it is not a real password I would make it explicit that it is a sensitive data. 
```suggestion
  password: REDACTED
```